### PR TITLE
Kev2 cluster import fix (integrate changes)

### PR DESCRIPTION
**Pull request by annablender**
Address https://github.com/rancher/rancher/issues/36128.

Update AKS, EKS, and GKE operator chart versions in rancher/charts

Fixes
* https://github.com/rancher/eks-operator/pull/59
* https://github.com/rancher/aks-operator/pull/52
* https://github.com/rancher/gke-operator/pull/73

### CODE REVIEW

File: packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 url: https://github.com/rancher/aks-operator/releases/download/v1.0.3/rancher-aks-operator-crd-1.0.3.tgz
```
Comment (kinarashah) [inline packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml:1]: We'd need to change this to `v1.0.4-rc1`, https://github.com/rancher/charts/pull/1625 for reference.

File: packages/rancher-aks-operator/rancher-aks-operator/package.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 url: https://github.com/rancher/aks-operator/releases/download/v1.0.3/rancher-aks-operator-1.0.3.tgz
-version: 100.0.2
+version: 100.0.3
```
Comment (thedadams) [inline packages/rancher-aks-operator/rancher-aks-operator/package.yaml:2]: Bumping these versions is correct, but the versions in the above URL also need to be bumped.

Same for all three operators.

**Review by thedadams — CHANGES_REQUESTED** [review]

**Review by thedadams — CHANGES_REQUESTED** [review]
Sorry, this part is new and I forgot about it. The new versions of the operators and their CRDs also need to be added to [release.yaml](https://github.com/raulcabello/charts/blob/dev-v2.6/release.yaml).

**Review by thedadams — DISMISSED** [review]

**Review by thedadams — DISMISSED** [review]

**Review by paynejacob — APPROVED** [review]

**Comment by annablender** [comment]
Fixed merge conflict. Changes are still the same

**Review by paynejacob — APPROVED** [review]

**Review by thedadams — CHANGES_REQUESTED** [review]
There is an extra commit in this PR that should be removed.

**Review by paynejacob — APPROVED** [review]

**Review by thedadams — APPROVED** [review]

**Pull request merged**